data.sql

```diff
@@ -4,3 +4,8 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon','11/15/2018',2,true,8);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu','01/07/2021',1,false,15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon','05/16/2017',5,true,11);
+
+-- Species
+alter table animals add column species varchar;
+
```

Comment:
- [ ] Please add this command in the schema.sql instead of data.sql file because schema file is responsible for data definition and data file is responsible for data manipulations and records :+1:

Reply:
As requested by the previous reviewer, you did remove this command from data.sql however, you didn't add it to the schema.sql file. Kindly do so and then ask for a re-review.